backend/user-service/tests/integration/user_test.go

```
@@ -61,19 +92,43 @@ func setupTestDatabase(db *sql.DB) error {
             is_active BOOLEAN DEFAULT true
         )
     `)
-    return err
+	return err
 }
 
 func cleanupTestDatabase(db *sql.DB) {
-    db.Exec("DROP TABLE IF EXISTS users")
+	db.Exec("DROP TABLE IF EXISTS users")
```

golangci-lint (1.64.8)

Error return value of `db.Exec` is not checked

(errcheck)